src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4

```diff
@@ -84,6 +84,7 @@ option
 
 optionKey
     : identifier
+    | STRING
```

Comment:
is this relevant?

Reply:
limits you to variable names only. e.g., you can't specify a spark config variable name as the option key (e.g., "rikai.model.config")

Reply:
as key, it should be an quanlifiedname , as `OPTIONS (rikai.model.config = 123)` but not a STRING `OPTIONS ("rikai.model.config" = 123)`?

Reply:
Filed #212